pkg/pool-weighted/test/foundry/ManagedPoolStorageLib.t.sol

```diff
@@ -37,11 +37,11 @@ contract ManagedPoolStorageLibTest is Test {
 
     uint256 private constant _MAX_SWAP_FEE = (1 << _SWAP_FEE_PCT_WIDTH) - 1;
 
-    function testSetJoinExitsEnabled(bytes32 poolState, bool enabled) public {
-        bytes32 newPoolState = ManagedPoolStorageLib.setJoinExitsEnabled(poolState, enabled);
+    function testsetJoinExitEnabled(bytes32 poolState, bool enabled) public {
```

Comment:
```suggestion
    function testSetJoinExitEnabled(bytes32 poolState, bool enabled) public {
```

Reply:
:eagle: :eye:

Reply:
Other tests (e.g., Liquidity Bootstrapping Pool) say `testSwapsEnabled`, but are calling `setSwapEnabled` on the mock.